movement/transforms.py

```diff
@@ -113,3 +116,155 @@ def scale(
     elif space_unit is None:
         scaled_data.attrs.pop("space_unit", None)
     return scaled_data
+
+
+def get_homography_transform(
```

Comment:
I wonder whether this should be named `compute_homography_transform`, to better match function names in our other modules.